docs/proto/events.md

```diff
@@ -117,44 +117,44 @@ Represents a security violation that is emitted by the agent
-| Violations | [string](#string) |  |  |
-| SubViolations | [string](#string) |  |  |
-| ViolationRating | [string](#string) |  |  |
-| SigSetNames | [string](#string) |  |  |
-| SigCVEs | [string](#string) |  |  |
-| ClientClass | [string](#string) |  |  |
-| ClientApplication | [string](#string) |  |  |
-| ClientApplicationVersion | [string](#string) |  |  |
-| Severity | [string](#string) |  |  |
-| ThreatCampaignNames | [string](#string) |  |  |
-| BotAnomalies | [string](#string) |  |  |
-| BotCategory | [string](#string) |  |  |
-| EnforcedBotAnomalies | [string](#string) |  |  |
-| BotSignatureName | [string](#string) |  |  |
-| ViolationsData | [ViolationData](#f5-nginx-agent-sdk-events-ViolationData) | repeated |  |
-| SystemID | [string](#string) |  |  |
-| InstanceTags | [string](#string) |  |  |
-| InstanceGroup | [string](#string) |  |  |
-| DisplayName | [string](#string) |  |  |
-| ParentHostname | [string](#string) |  |  |
+| PolicyName | [string](#string) |  | The name of the policy that triggered the security violation |
+| SupportID | [string](#string) |  | The unique support ID of the violation to be tracked with |
+| Outcome | [string](#string) |  | The outcome that resulted for the security violation |
+| OutcomeReason | [string](#string) |  | The reason for the security violation to result into the outcome |
+| BlockingExceptionReason | [string](#string) |  | The reason why the request was blocked |
+| Method | [string](#string) |  | The HTTP Method of the request that triggered the security violation |
+| Protocol | [string](#string) |  | The HTTP Protocol of the request that triggered the security violation |
+| XForwardedForHeaderValue | [string](#string) |  | The HTTP xff_header_value of the request that triggered the security violation |
+| URI | [string](#string) |  | The URI of the request that triggered the security violation |
+| Request | [string](#string) |  | The full request that triggered the security violation including the METHOD URI and request Body |
+| IsTruncated | [string](#string) |  | If the request is truncated or not |
+| RequestStatus | [string](#string) |  | The status of the request that triggered the security violation |
+| ResponseCode | [string](#string) |  | The response code to the request that triggered the security violation |
+| ServerAddr | [string](#string) |  | The server address of the instance that caught the security violation |
+| VSName | [string](#string) |  | The VS Name of the instance that caught the security violation |
+| RemoteAddr | [string](#string) |  | The targeted address by the request that triggered the security violation |
+| RemotePort | [string](#string) |  | The targeted port number by the request that triggered the security violation |
+| ServerPort | [string](#string) |  | The server port of the instance that caught the security violation |
+| Violations | [string](#string) |  | A comma separated list of all the violations triggered by the request |
```

Comment:
nit hyphenation: 
```suggestion
| Violations | [string](#string) |  | A comma-separated list of all the violations triggered by the request |
```
Same applies for all instances of 'comma separated' here.

Reply:
applied 👍